src/libstd/keyword_docs.rs

```diff
@@ -895,6 +895,15 @@ mod mod_keyword {}
 /// // x is no longer available
 /// ```
 ///
+/// `move` is also valid before an async block.
+///
+/// ```rust
+/// let capture = "hello";
+/// async move {
```

Comment:
In the change to move to async blocks here, it looks like the let binding was also removed-- I'd leave that in, since this block on its own won't cause any code to run.